packages/analytics-js/src/services/ErrorHandler/types.ts

```diff
@@ -0,0 +1,12 @@
+export type ErrorState = {
+  severity: string;
+  unhandled: boolean;
+  severityReason: { type: string };
+};
+
+export type EventTarge = {
```

Comment:
what is the need for a custom type instead of the default one for event.target?

Reply:
event.target is alright but event.target.dataset, event.target.localName is not there is the type that is why I added

Reply:
(event.target as Element).localName exists?

Reply:
Adressed